pt/lessons/basics/modules.md

```diff
@@ -207,18 +207,69 @@ Se tentar chamar um macro que ainda não está carregado, Elixir vai gerar um er
 
 ### `use`
 
-Usa o módulo no contexto atual. Isto é particularmente útil quando um modulo que precisa executar alguma configuração. Ao chamar `use` nós invocamos o `__using__` dentro do modulo, proporcionando o módulo uma oportunidade para modificar nosso contexto existente:
+Com a macro `use` podemos dar habilidade a outro módulo para modificar a sua definição atual.
+Quando invocamos `use` em nosso código estamos invocando o callback `__using__/1` definido pelo módulo declarado.
+O resultado da macro `__using__/1` passa a fazer parte da definição do módulo.
+Para melhor entendimento de como isso funciona vamos olhar um exmplo simples:
 
 ```elixir
-defmodule MyModule do
-  defmacro __using__(opts) do
+defmodule Hello do
+  defmacro __using__(_opts) do
     quote do
-      import MyModule.Foo
-      import MyModule.Bar
-      import MyModule.Baz
+      def hello(name), do: "Hi, #{name}"
+    end
+  end
+end
+```
+
+Aqui criamos um módulo `Hello` que define o callback `__using__/1`, o qual define a função `hello/1`.
+Vamos criar um novo módulo para tentar nosso novo código:
+
+```elixir
+defmodule Example do
+  use Hello
+end
+```
+
+Se executarmos nosso código no IEx veremos que `hello/1` está disponível no módulo `Example`.
+
+```elixir
+iex> Example.hello("Sean")
+"Hi, Sean"
+```
+
+Aqui podemos ver que `use` invoca o callback `__using__/1` dentro do `Hello` o qual acaba adicionando o código dentro do nosso módulo.
+Agora que demonstramos o exemplo básico vamos atualizar nosso código para entender como `__using__/1` suporta opções.
+Vamos fazer isso adicionando a opção `greeting`:
 
-      alias MyModule.Repo
+```elixir
+defmodule Hello do
+  defmacro __using__(opts) do
+    greeting = Keyword.get(opts, :greeting, "Hi")
+
+    quote do
+      def hello(name), do: unquote(greeting) <> ", " <> name
     end
   end
 end
 ```
+
+Vamos atualizar nosso módulo `Example` para incluir a opção nova `greeting`:
+
+```elixir
+defmodule Example do
+  use Hello, greeting: "Hola"
+end
+```
+
+Se executamos isso dentro do IEx devemos ver que a função greeting foi alterada:
+
+```
+iex> Example.hello("Sean")
+"Hola, Sean"
+```
+
+Esses exemplos simples demonstram como `use` funciona, mas essa é uma ferramenta poderosa na caixa de ferramentas do Elixir.
+À medida que você passa a aprender mais sobre Elixir, observe bem como o use é utilizado. Um exemplo que você verá com certeza é use ExUnit.Case, async: true.
```

Comment:
Just a final observation: I think you unintentionally removed the apostrophes in `use` and `use ExUnit.Case, async: true`.

Reply:
copy paste issue rs